pkg/openstack/loadbalancer.go

```diff
@@ -1711,9 +1708,13 @@ func (lbaas *LbaasV2) ensureOctaviaLoadBalancer(ctx context.Context, clusterName
 		}
 	}
 
-	addr, err := lbaas.ensureFloatingIP(clusterName, service, loadbalancer, svcConf, isLBOwner)
-	if err != nil {
-		return nil, err
+	addr := loadbalancer.VipAddress
```

Comment:
I'm a bit uneasy of using `VipAddress` here because of [1]. I think what I'd do is to make sure we're explicitly logging and maybe adding an event about skipping FIP creation.

I also think that code fragment [1] needs a change to make sure we're supporting LoadBalancerIP enforcement for IPv6.

[1] https://github.com/kubernetes/cloud-provider-openstack/blob/8245f5385c4845f54df8bf95f8caec571aa7fc5d/pkg/openstack/loadbalancer.go#L260-L264

Reply:
I just hit an internal server error when using a IPv6 from the nodes subnet to as a loadbalancerIP. The subnet was configured with slaac. I will try with status to see if there is any difference

Reply:
done. Note that this will only work with addresses that were generated from a stateful subnet and not slaac.
Otherwise we would hit the following issue: `cannot be directly assigned to a port on subnet a215f064-5fe6-4a15-856a-a7e70912e510 as the subnet is configured for automatic addresses`